docs/source/ingest/destination_connectors/pinecone.rst

```diff
@@ -0,0 +1,80 @@
+Pinecone
+===========
+
+Batch process all your records using ``unstructured-ingest`` to store structured outputs and embeddings locally on your filesystem and upload those to a Pinecone index.
+
+First you'll need to install the Pinecone dependencies as shown here.
+
+.. code:: shell
+
+  pip install "unstructured[pinecone]"
+
+Run Locally
+-----------
+The upstream connector can be any of the ones supported, but for convenience here, showing a sample command using the
+upstream s3 connector. This will create new files on your local.
+
+.. tabs::
+
+   .. tab:: Shell
+
+      .. code:: shell
+
+        unstructured-ingest \
+            s3 \
+            --remote-url s3://utic-dev-tech-fixtures/small-pdf-set/ \
+            --anonymous \
+            --output-dir s3-small-batch-output-to-pinecone \
+            --num-processes 2 \
+            --verbose \
+            --strategy fast \
+            --chunk-elements \
+            --embedding-provider <an unstructured embedding provider, ie. langchain-huggingface> \
+            pinecone \
+            --api-key <your pinecone api key here> \
+            --index-name <your index name here, ie. ingest-test> \
+            --environment <your environment name here, ie. gcp-starter> \
+            --batch-size <number of elements to be uploaded per batch, ie. 80> \
+            --num-processes <number of processes to be used to upload, ie. 2>
+
+   .. tab:: Python
+
+      .. code:: python
+
+        import subprocess
+
+        command = [
```

Comment:
This should also be using the pythonic approach to running the code, example: [mongodb.rst](https://github.com/Unstructured-IO/unstructured/blob/roman/mongo-db-atlas-dest-conn/docs/source/ingest/destination_connectors/mongodb.rst?plain=1#L45)

Reply:
Agreed, addressed with [pythonic approach in docs](https://github.com/Unstructured-IO/unstructured/pull/1774/commits/cbd734f4303c76884228f858763c52ca21be6b38)